src/components/radio-group/radio-group.tsx

```diff
@@ -124,6 +130,10 @@ export class RadioGroup implements LabelableComponent, FormComponent, Interactiv
     }
   }
 
+  componentDidLoad(): void {
+    afterConnectDefaultValueSet(this, this.value);
```

Comment:
I feel like a more intuitive name for this would be `setDefaultValueAfterConnect`

Reply:
lol

Reply:
After connect default value you set